### DISCUSSION TITLE
How do I add shellcheck arguments?

### DISCUSSION THREAD

**Discussion by txtsd** [LazyVim/LazyVim#4703] [Q&A] [answered]
I run shellcheck like this `shellcheck -s bash -e SC2164,SC2034,SC2154`\
and I want to configure lazyvim/nvim-lint to use those arguments.

**Comment by dotfrag** [comment, marked as answer]
Either look into [Customize built-in linters](https://github.com/mfussenegger/nvim-lint?tab=readme-ov-file#customize-built-in-linters) or use a global shellcheck rc file. See [Ignoring one or more types of errors forever](https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-one-or-more-types-of-errors-forever). Despite the docs saying "As of v0.7.0 you can create a file .shellcheckrc in your home directory", I have mine in `~/.config/shellcheckrc` and it works just fine. Maybe the documentation is not updated.

**Comment by txtsd** [comment]
Thanks! A global shellcheck file actually works for me as a solution.
